nemo/nemo/utils/misc.py

```diff
@@ -1,3 +1,4 @@
+# Copyright (c) 2019 NVIDIA Corporation
 from collections import UserDict
 
```

Comment:
Analogical comment as previously - do we really need files that contain a single line of import?